### PR TITLE
add simple deref macro

### PR DESCRIPTION
**Pull request by vhdirk**
This adds a really simple macro for the deref impl

### CODE REVIEW

File: gobject-subclass/src/deref.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
+#[macro_export]
+macro_rules! gobject_subclass_deref(
+
```
Comment (sdroege) [inline gobject-subclass/src/deref.rs]: An empty line to remove :)

**Comment by sdroege** [comment]
Looks generally good, thanks!

**Comment by sdroege** [comment]
```
error[E0599]: no method named `starts_with` found for type `std::ffi::OsString` in the current scope
  --> gio-subclass/tests/simple_application.rs:95:30
   |
95 |                 assert!(!arg.starts_with("--local-"))
   |                              ^^^^^^^^^^^
```

This one is weird. Also you can get rid of the `std::ops` imports now

**Comment by vhdirk** [comment]
That error is odd indeed. I didn't even touch that part.

**Comment by vhdirk** [comment]
That error seems to stem from this:
https://github.com/gtk-rs/gio/commit/efc6c902b6198c02282254dbe2ea891311b70c23

**Comment by vhdirk** [comment]
#8 fixes the tests

File: gobject-subclass/src/deref.rs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
```
Comment (sdroege) [inline gobject-subclass/src/deref.rs]: Unnecessary empty line

**Comment by sdroege** [comment]
Except for the empty line, this seems good to go :) Remove that and merge, thanks!

**Pull request merged**